scripts/reconciliation.py

```diff
@@ -42,26 +42,26 @@
 # check specific streams [corgi ofuri, deptopia id]
 stream_ga10_corpus = [
     ["o:redhat:ansible_automation_platform:1.2", "2085"],  # GA 1.0
-    ["o:redhat:ansible_automation_platform:2.0", "2030"],  # GA 1.0
-    ["o:redhat:ansible_automation_platform:2.1", "2154"],  # GA 1.0
-    ["o:redhat:ansible_automation_platform:2.2", "2340"],  # GA 1.0
-    ["o:redhat:openshift:4.8.z", "103"],  # GA 1.0
-    ["o:redhat:openshift:4.9.z", "100"],  # GA 1.0
-    ["o:redhat:openshift:4.10.z", "2305"],  # GA 1.0
-    ["o:redhat:openshift:4.11.z", "2367"],  # GA 1.0
-    ["o:redhat:rhacm:2.3.z", "104"],  # GA 1.0
-    ["o:redhat:rhacm:2.4.z", "1444"],  # GA 1.0
-    ["o:redhat:rhacm:2.5.z", "2397"],  # GA 1.0
-    ["o:redhat:rhacm:2.6.z", "2394"],  # GA 1.0
-    ["o:redhat:rhacm:2.7", "2395"],  # GA 1.0
-    ["o:redhat:rhel:8.4.0.z", "35"],  # GA 1.0
-    ["o:redhat:rhel:8.6.0.z", "2302"],  # GA 1.0
-    ["o:redhat:rhel:8.7.0", "2301"],  # GA 1.0
-    ["o:redhat:rhel:9.0.0.z", "2307"],  # GA 1.0
-    ["o:redhat:rhel:9.1.0", "2308"],  # GA 1.0
-    ["o:redhat:rhn_satellite:6.7", "93"],  # GA 1.0
-    ["o:redhat:rhn_satellite:6.8", "50"],  # GA 1.0
-    ["o:redhat:rhn_satellite:6.9", "72"],  # GA 1.0
+    # ["o:redhat:ansible_automation_platform:2.0", "2030"],  # GA 1.0
```

Comment:
I do not think you meant to commit this change

Reply:
Thanks, I removed changes to the reconciliation script from this PR.